actions/post_actions.jsx

```diff
@@ -31,72 +44,108 @@ const dispatch = store.dispatch;
 const getState = store.getState;
 
 export function handleNewPost(post, msg) {
-    let websocketMessageProps = {};
-    if (msg) {
-        websocketMessageProps = msg.data;
-    }
+    return async (doDispatch, doGetState) => {
```

Comment:
should this be moved to the `actions/views/posts.js` file? and why does this have websocket msg props? in that case should be a WebSocket action, right?

Reply:
As discussed offline, we leave it as is right now.
Also, `handleNewPost` is also used for some ephemeral post generated by the webapp.